modules/bioinspired/include/opencv2/bioinspired/retina.hpp

```diff
@@ -438,13 +438,13 @@ class CV_EXPORTS_W Retina : public Algorithm {
     @param reductionFactor only usefull if param useRetinaLogSampling=true, specifies the reduction
     factor of the output frame (as the center (fovea) is high resolution and corners can be
     underscaled, then a reduction of the output is allowed without precision leak
-    @param samplingStrenght only usefull if param useRetinaLogSampling=true, specifies the strenght of
+    @param samplingStrentgh only usefull if param useRetinaLogSampling=true, specifies the strength of
```

Comment:
Wrong parameter name in documentation here.

Reply:
@mshabunin I fixed this point in https://github.com/opencv/opencv_contrib/pull/2412/commits/54b0ea3e4719a2f2feaaeef3058518d91b6aeb58.